docs/lunaria.config.json

```
@@ -0,0 +1,94 @@
+{
+  "$schema": "./node_modules/@lunariajs/core/config.schema.json",
+  "repository": {
+    "name": "withastro/starlight",
+    "rootDir": "docs"
+  },
+  "defaultLocale": {
+    "label": "English",
+    "lang": "en"
+  },
+  "locales": [
+    {
+      "label": "Dansk",
+      "lang": "da"
+    },
+    {
+      "label": "Deutsch",
+      "lang": "de"
+    },
+    {
+      "label": "Español",
+      "lang": "es"
+    },
+    {
+      "label": "Français",
+      "lang": "fr"
+    },
+    {
+      "label": "हिंदी",
+      "lang": "hi"
+    },
+    {
+      "label": "Bahasa Indonesia",
+      "lang": "id"
+    },
+    {
+      "label": "Italiano",
+      "lang": "it"
+    },
+    {
+      "label": "日本語",
+      "lang": "ja"
+    },
+    {
+      "label": "한국어",
+      "lang": "ko"
+    },
+    {
+      "label": "Português do Brasil",
+      "lang": "pt-br"
+    },
+    {
+      "label": "Русский",
+      "lang": "ru"
+    },
+    {
+      "label": "Türkçe",
+      "lang": "tr"
+    },
+    {
```

We recently merged in Ukrainian too!

```suggestion
    {
      "label": "Українська",
      "lang": "uk"
    },
    {
```